src/web/client/extension.ts

```diff
@@ -92,7 +92,7 @@ export function activate(context: vscode.ExtensionContext): void {
                         );
                     }
                 }
-                const geo = queryParamsMap.get('geo')?.toLowerCase();
+                const geo = queryParamsMap.get(queryParameters.ORG_GEO)?.toLowerCase() ?? 'us';
```

Comment:
is defaulting to US a good idea here? Specially given we can have EU org here?

Reply:
I will check once with Dileep on this but this will act as unauthenticated scenario incase orgGeo is empty. As of now we have decided for all unauthenticated scenario will pass the logging to US

Reply:
Integrated Artemis service response